src/pages/LatestPosts/LatestPost.style.js

```diff
@@ -2,13 +2,19 @@ import palette from '../../assets/colors';
 
 const latestPostStyle = (theme) => ({
     width: 'calc(50% - 5px)',
-    '@media (max-width: 767px)':{width: '100%'},
     padding: '20px',
     marginBottom: '10px',
     borderRadius: '20px',
     boxShadow: `0px 6px 26px ${palette[theme].shadowColor}`,
-    ':nth-child(2n+1)': {
-      marginRight: '10px'
+
+    '@media (min-width: 768px)':{
+      ':nth-child(2n+1)': {
+        marginRight: '10px'
+      }
+    },
+   
```

Comment:
Please, rewrite it using [gap](https://caniuse.com/flexbox-gap) property for the parent component. On another hand, You can aslo use well-known `flex` property for the children

Reply:
Pls, look through https://habr.com/ru/post/468327/